lab2/part1/myrouter_part1.py

```diff
@@ -0,0 +1,149 @@
+'''
+Title:          myrouter_part1
+Description:    Project 2, part 1 - implement basic ARP handling in a simulated router
+Authors:        David Billmire, Shang Ding
+Course:         UWCS 640, Section 2, Fall 2019
+'''
+
+import sys
+import os
+import time
+
+from switchyard.lib.packet.util import *
+from switchyard.lib.userlib import *
+
+'''
+Class:          ARPContext
+Description:    Implements basic ARP functions, including IP to MAC translation and table management
+                ARP Context is iterable and supports direct get/set of IPv4 -> MAC mapping
+'''
+class ARPContext:
+    def __init__(self):
+        self.map = {}
+        return
+    
```

Comment:
There are trailing space characters here that should be trimmed. And there are more occurrences of this on line 93, 125, and etc.

Reply:
I used yapf in lab 1 for formatting, and I think it worked well.

https://github.com/google/yapf